Reflections/Week_6_reflection.md

```diff
@@ -0,0 +1,20 @@
+## Accomplishments
+
+I have begun doing user testing for my program.
+As well as making a proper windows build for my program to allow more users to test.
+
+## Challenges
+
+I had to go through the complicated and difficult process of working out issues in my build chain. This was eventually resolved though.
+
+## Next Steps
+
+I have obtained a number of glitches that need resolving in my code from user testing with a friend.
+
+I want to first add them all into my issue tracker so that I can triage them, this should take an hour or so.
+Then I will focus first on resolving the issue where saving and loading breaks tool selection, I think this will take a few days.
+Finally I want to do some quality of life improvmenents, Such as un-inverting scroll. improving the selection bounding boxes, and other issues I have notes for. This will probably take a while to iron all the kinks out.
```

Comment:
Great SMART goals here! Specificity on the duration of the last goal would be effective, but I'm pickin' up what you're puttin' down.